src/templates/react/Content/src/App.fs

```diff
@@ -66,11 +66,15 @@ let root model dispatch =
                     [ pageHtml model.currentPage ] ] ] ] ]
 
 open Elmish.React
+#if DEBUG
 open Elmish.Debug
```

Comment:
Not only that, Fable won't include any reference to the file if you don't actually use anything from there (opening is just not enough). It could be argued that this is a bug because you may not get side effects when opening a module, but unfortunately the F# AST doesn't give the open statements, only the actual calls.

Anyways, TL;DR; Yes, it's not necessary to add `#if DEBUG` around open statements.